app/models/file_search.rb

```diff
@@ -98,11 +99,12 @@ def search_query
     query[:q] = search_term if search_term
     query[:f] = filters unless filters.empty?
     query[:page] = page if page
+    query[:per_page] = per_page
     query
   end
 
   def has_query_params?
-    (params.stringify_keys.keys & %w(q work venues years types)).any?
+    (params.stringify_keys.keys & %w(q work venues years)).any?
```

Comment:
Why did `types` go away here?  I thought `types` was the filter selections and `resource_type` is the thing we're looking for.